### PR TITLE
Add new condition operation: not-has-key

### PR DESCRIPTION
**Pull request by Era-cell**

### CODE REVIEW

File: DSL/conditions.md [hunk 1]
```diff
@@ -212,6 +212,27 @@ Here's a detailed explanation:
...
+`not-has-key`: Checks if the specified key is absent in the target object.
+
+- **Value**: Must be a string.
+
+##### Schema
+
+```json
+{
+  "$schema": "https://json-schema.org/draft-03/schema",
+  "divisibelBy": 3
```
Comment (jviotti) [inline DSL/conditions.md:224]: There is a typo here

File: DSL/conditions.md [hunk 2]
```diff
@@ -212,6 +212,27 @@ Here's a detailed explanation:
...
+}
+```
+
+##### Condition
+
+```json
+[
+  { "path": [ ], "operation": "not-has-key", "value": "multipleOf" }
```
Comment (jviotti) [inline DSL/conditions.md:232]:
```suggestion
  { "path": [], "operation": "not-has-key", "value": "multipleOf" }
```

**Review by jviotti — APPROVED** [review]
A couple comments, but looks good once those are addressed

**Comment by Era-cell** [comment]
> A couple comments, but looks good once those are addressed

Done 👍, can I myself merge ?

**Comment by jviotti** [comment]
Yeah, feel free to self-merge in general for approved PRs!

**Pull request merged**